SU-HSDO/suhumsci
# Feat 109  2 3 column views

# READY FOR REVIEW

## Summary
Add support for 2 and 3 column views by using Drupal's default "Grid" options, as shown below.

![image](https://user-images.githubusercontent.com/1256329/73279323-f4806880-41ba-11ea-9e28-65cae47f6b5d.png)


## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Set up a view, using the "grid" format, test using 2 and 3 columns, and confirm that the content displays in the number of columns configured and scales down appropriately on smaller screen widths.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
